### PR TITLE
Make one device pass and one device fail in FOT test

### PR DESCRIPTION
**Pull request by didovesei**

### CODE REVIEW

File: controller/bin/daq_system_default.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
```
Comment (grafnu) [inline controller/bin/daq_system_default.yaml:1]: why pull in a copy of this file?

File: controller/bin/daq_config.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # DAQ FOT configuration.
 
-include: ../daq/config/system/default.yaml
+include: daq_system_default.yaml
```
Comment (grafnu) [inline controller/bin/daq_config.yaml:3]: why create a new separate file?

File: bin/setup_stack [hunk 3]
```diff
@@ -178,7 +184,7 @@ fi
 
 echo Connecting faux interfaces...
 add_faux t2sw1 1 1 $faux_dhcp
-add_faux t2sw2 1 2 $faux_dhcp
+add_faux t2sw2 1 2 $faux_dhcp $faux_http
```
Comment (grafnu) [inline bin/setup_stack:187]: can this just be fixed?  Does anything bad happen if we just always enable http on this device?  I'm trying to keep the overall complexity down unless there's a specific reason to make it configurable!

File: etc/DAQ_VERSION [hunk 4]
```diff
@@ -1 +1 @@
-1.9.13
+master
```
Comment (grafnu) [inline etc/DAQ_VERSION:1]: Is there a specific DAQ version/change you need?  It's trivial for us now to tag an intermediate DAQ version (e.g. 1.9.14) -- so we can just do that if necessary.

File: etc/start_faux [hunk 5]
```diff
@@ -53,5 +53,10 @@ if [ -n "${options[dnsmasq]}" ]; then
     dnsmasq --log-facility=/tmp/dnsmasq.log --log-dhcp
 fi
 
+if [ -n "${options[http]}" ]; then
+    echo Listening on TCP 80...
+    python3 -m http.server &
```
Comment (henry54809) [inline etc/start_faux:58]: This by default listens on port 8000.

File: controller/bin/daq_config.yaml (hunk 2 above)
Comment (didovesei) [reply controller/bin/daq_config.yaml]: Reverted. I had some path issue with some DAQ version but now it is working. Not sure what fixed it...

File: bin/setup_stack (hunk 3 above)
Comment (didovesei) [reply bin/setup_stack]: Done. The http server is always running on faux-2 now.

File: etc/DAQ_VERSION (hunk 4 above)
Comment (didovesei) [reply etc/DAQ_VERSION]: Using 1.9.14 now

File: etc/start_faux (hunk 5 above)
Comment (didovesei) [reply etc/start_faux:58]: Updated the echo message. I saw daq does fail the device even if it is listing on 8000.

File: controller/bin/daq_system_default.yaml (hunk 1 above)
Comment (didovesei) [reply controller/bin/daq_system_default.yaml]: Removed

**Review by grafnu — APPROVED** [review]

**Pull request merged**